packages/api/src/schema/api-schema.yaml

```diff
@@ -0,0 +1,3102 @@
+                    type: string
+        - type: object
+          additionalProperties: false
+          required:
+            - ipfs
+          properties:
+            ipfs:
+              type: object
+              additionalProperties: false
+              properties:
+                $ref: >-
+                  #/components/schemas/asset/properties/storage/properties/ipfs/properties/spec/properties
+                pinata:
+                  description: >-
+                    Custom credentials for the Piñata service. Must have either
+                    a JWT or an API key and an API secret.
+                  oneOf:
+                    - type: object
+                      additionalProperties: false
+                      required:
+                        - jwt
+                      properties:
+                        jwt:
+                          type: string
+                          writeOnly: true
+                          description: >-
+                            Will be added to the Authorization header as a
+                            Bearer token.
+                    - type: object
+                      additionalProperties: false
+                      required:
+                        - apiKey
+                        - apiSecret
+                      properties:
+                        apiKey:
+                          type: string
+                          description:
+                            Will be added to the pinata_api_key header.
+                        apiSecret:
+                          type: string
+                          writeOnly: true
+                          description:
+                            Will be added to the pinata_secret_api_key header.
+    signing-key:
+      type: object
+      additionalProperties: false
+      required:
+        - publicKey
+      properties:
+        id:
+          type: string
+          example: 78df0075-b5f3-4683-a618-1086faca35dc
+        name:
+          type: string
+          description: Name of the signing key
+        createdAt:
+          type: number
+          description:
+            Timestamp (in milliseconds) at which the signing-key was created
+          example: 1587667174725
+        lastSeen:
+          type: number
+          description:
+            Timestamp (in milliseconds) at which the signing-key was last used
+          example: 1587667174725
+        userId:
+          type: string
+          example: 78df0075-b5f3-4683-a618-1086faca35dc
+        publicKey:
+          type: string
+        deleted:
+          type: boolean
+          default: false
+        disabled:
+          type: boolean
+          description: Disable the signing key to allow rotation safely
+    signing-key-response-payload:
+      type: object
+      required:
+        - publicKey
+        - privateKey
+      properties:
+        $ref: "#/components/schemas/signing-key/properties"
+        privateKey:
+          type: string
```

Comment:
So this is a little hack we've used in our schema: a `$ref` key that is not the only entry in the object, and then we add more keys to it. Make sure this works with readme.io!